server/util/timeutil/timeutil.go

```diff
@@ -0,0 +1,12 @@
+package timeutil
```

Comment:
no good deed goes unpunished.... can you send a followup CL that uses this library for all the conversions we do to and from time.Time from usec?

Reply:
lol, will do